src/OrchardCore/OrchardCore.Environment.Shell.Abstractions/Builders/ShellContext.cs

```diff
@@ -58,75 +65,98 @@ public IServiceScope EnterServiceScope()
         /// </summary>
         public int ActiveRequests => _refCount;
```

Comment:
Maybe we could remove this property or rename it as `ActiveScopes` and update the comment.